packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/error/NoPrefferedPackageFound.java

```diff
@@ -0,0 +1,10 @@
+package expo.modules.webbrowser.error;
+
+import org.unimodules.core.errors.CodedException;
+
+public class NoPrefferedPackageFound extends CodedException {
```

Comment:
How about also overriding `getCode()` and using plain `promise.reject(exception)` in the module?